ci/hera_cal_tests.yml

```diff
@@ -27,8 +27,7 @@ dependencies:
   # - pyuvdata==2.2.12
 
   - pip:
-    - git+https://github.com/RadioAstronomySoftwareGroup/pyuvdata
-    # - git+https://github.com/HERA-Team/aipy
+    - git+https://github.com/RadioAstronomySoftwareGroup/pyuvdata@cal-init-method
```

Comment:
I don't love pulling in this PR while the necessary software is still on a branch over at pyuvdata... any chance that'll get pulled in first?

Reply:
That PR is in now, so I've updated this.